pwiz_tools/Skyline/FileUI/PeptideSearch/PeptideSearchResources.resx

```diff
@@ -362,4 +362,7 @@ Click 'Retry' to try building again after placing the original spectrum files in
   <data name="ImportPeptideSearchDlg_NextPage_Adding_detected_features_to_document" xml:space="preserve">
     <value>Adding detected features to document</value>
   </data>
+  <data name="SearchControl_CanWizardClose_Cannot_close_wizard_while_the_search_is_running_" xml:space="preserve">
+      <value>Cannot close wizard while the search is running. Press the Cancel Search button first.</value>
```

Comment:
Should go the extra step of asking the user if they want to cancel rather than just telling them they need to press the button themselves. You can still deny the close until the cancel is completed, but it is more polite to help the user do what you are telling them is necessary.